src/Appwrite/Migration/Version/V19.php

```diff
@@ -706,35 +705,38 @@ protected function fixDocument(Document $document): Document
                 }
 
                 $commands = $this->getFunctionCommands($document);
-                $document->setAttribute('commands', $commands);
-
-                $schedule = $this->consoleDB->createDocument('schedules', new Document([
-                    'region' => App::getEnv('_APP_REGION', 'default'), // Todo replace with projects region
-                    'resourceType' => 'function',
-                    'resourceId' => $document->getId(),
-                    'resourceInternalId' => $document->getInternalId(),
-                    'resourceUpdatedAt' => DateTime::now(),
-                    'projectId' => $this->project->getId(),
-                    'schedule'  => $document->getAttribute('schedule'),
-                    'active' => !empty($document->getAttribute('schedule')) && !empty($document->getAttribute('deployment')),
-                ]));
-
-                $document->setAttribute('scheduleId', $schedule->getId());
-                $document->setAttribute('scheduleInternalId', $schedule->getInternalId());
+                $document->setAttribute('commands', $document->getAttribute('commands', $commands));
+
+                if ($document->getAttribute('schedule') && !$document->getAttribute('scheduleId', null)) {
```

Comment:
schedule doesn't matter when determining whether a schedule document should exist:

```suggestion
                if (empty($document->getAttribute('scheduleId', null))) {
```